### DISCUSSION TITLE
[PROPOSAL] Approach for triaging Sherpa repo issues

### DISCUSSION THREAD

**Discussion by oshoma** [Aggregate-Intellect/sherpa#328] [General]
Let's triage the issues list so that we can focus better.  As of Nov 10 we have 42 issues. Some appear to be duplicates, and should be merged. Some are not labeled, and should be. There may also be some issues which will never be fixed, and if so, these should be labeled "won't fix" and closed.

**Comment by oshoma** [comment]
We discussed our approach for this in the sherpa-dev meeting today. Here's the proposal so far:

1. Periodically triage the issues list so that we touch every open issue at least once every N days (N = 30?)
2. An issue that has received no updates in X days will be closed/archived (X = 90? 120?)
3. Add labels to issues  e.g. `duplicate`. We might need some more labels.
4. Post project priorities on Slack and maybe on Github too to give people a better sense of where we are headed 


Next steps:

- [x] @bharatji30 will go through and identify potential duplicates
- [x] All review and comment here on this proposal
- [ ] @oshoma review issues again and suggest a draft prioritization for the group to review in future

**Comment by bharatji30** [comment]
[Sherpa issues.xlsx](https://github.com/Aggregate-Intellect/sherpa/files/13425294/Sherpa.issues.xlsx)

Here is the consolidation of the issues as of 11/12/23. We can discuss them more in the triage discussion.

**Comment by oshoma** [comment]
Thanks @bharatji30 this is very helpful. Much appreciated!
Thanks @20001LastOrder for labeling more issues.
Do either of you have any feedback on my proposal above? The 4 steps I suggested last week.

**Comment by 20001LastOrder** [comment]
@oshoma I think the 4 steps are a great start. Thanks! For step 2, maybe we can set up a bot to automate the issue closing. And I think we should think about step 4 sooner or later. This may help people find issues to work on.
